pages/linux/distrobox.md

```diff
@@ -0,0 +1,41 @@
+# distrobox
```

Comment:
I think this is a case similar to https://github.com/tldr-pages/tldr/blob/main/pages/common/transmission.md, but the command and the page name should match. and this is not the case.

Reply:
Okay, I'll seperate the doc into different docs.